### ISSUE TITLE
Ansible 2.4.0 compatible?

### ISSUE THREAD

**Issue by mf69** [willthames/ansible-inventory-grapher#30]
ansible-inventory-grapher (2.4.0) seems to not work with ansible 2.4.0, for me anyway.
```
$ /usr/local/bin/ansible-inventory-grapher --ask-vault-pass -i production -d graphs/ -q hostname
Traceback (most recent call last):
  File "/usr/local/bin/ansible-inventory-grapher", line 182, in <module>
    main()
  File "/usr/local/bin/ansible-inventory-grapher", line 178, in main
    render_graph(arg, options)
  File "/usr/local/bin/ansible-inventory-grapher", line 125, in render_graph
    vault_pass = ask_vault_pass()
  File "/usr/local/bin/ansible-inventory-grapher", line 36, in ask_vault_pass
    return utils.ask_passwords(ask_vault_pass=True)[2]
NameError: global name 'utils' is not defined
```

Can anyone reproduce this issue?
```
$ pip list | grep ansible
ansible (2.4.0.0)
ansible-inventory-grapher (2.4.0)
```

**Comment by willthames** [comment]
You're quite right, this is currently a bug. Fixing the error shown here is pretty straightforward, but ansible 2.4 contains a massive rewrite of the whole inventory stack and I haven't had time to work out whether to write a compatibility layer or extend `ansible-inventory` that now comes with ansible to provide more of the functionality available in the grapher.

The easiest workaround is to run ansible-inventory-grapher in an ansible-2.3 virtualenv

```
virtualenv /path/to/my/venv
source /path/to/my/venv/bin/activate
pip install 'ansible<2.4' ansible-inventory-grapher
```

**Comment by willthames** [comment]
I've finally managed to devote sufficient time to this.

I've pushed the latest version to the ansible-2.4 branch for testing. If you're able to test, let me know - I'll likely push it to pypi in the next few days either way.

**Comment by mf69** [comment]
Happily I will test, have you some basic instructions to follow?

**Comment by mf69** [comment]
```
$ pip list | grep ansible
ansible (2.4.1.0)
ansible-inventory-grapher (2.4.2)

$ /usr/local/bin/ansible-inventory-grapher --ask-vault-pass -i production -d graphs/ -q kvm-servers
Traceback (most recent call last):
  File "/usr/local/bin/ansible-inventory-grapher", line 154, in <module>
    main()
  File "/usr/local/bin/ansible-inventory-grapher", line 150, in main
    render_graph(arg, options)
  File "/usr/local/bin/ansible-inventory-grapher", line 109, in render_graph
    inventory_mgr = ansibleinventorygrapher.inventory.InventoryManager(options.inventory, options.ask_vault_pass, options.vault_password_file)
  File "/usr/local/lib/python2.7/dist-packages/ansibleinventorygrapher/inventory.py", line 158, in __init__
    self.inventory = INVENTORY(inventory, ask_vault_pass, vault_password_file)
  File "/usr/local/lib/python2.7/dist-packages/ansibleinventorygrapher/inventory.py", line 104, in __init__
    super(Inventory24, self).__init__(inventory, ask_vault_pass, vault_password_file)
  File "/usr/local/lib/python2.7/dist-packages/ansibleinventorygrapher/inventory.py", line 24, in __init__
    self.vault_pass = self.ask_vault_password()
  File "/usr/local/lib/python2.7/dist-packages/ansibleinventorygrapher/inventory.py", line 98, in ask_vault_password
    from ansible.parsing.vault import PromptVaultPass
ImportError: cannot import name PromptVaultPass
```

**Comment by barbudone** [comment]
samo-samo arround here:

ansible-inventory-grapher --ask-vault-pass -i /etc/ansible/hosts asus -d test-inv --format "test-asus.dot" 
Traceback (most recent call last):
  File "/home/user/.local/bin/ansible-inventory-grapher", line 154, in <module>
    main()
  File "/home/user/.local/bin/ansible-inventory-grapher", line 150, in main
    render_graph(arg, options)
  File "/home/user/.local/bin/ansible-inventory-grapher", line 109, in render_graph
    inventory_mgr = ansibleinventorygrapher.inventory.InventoryManager(options.inventory, options.ask_vault_pass, options.vault_password_file)
  File "/home/user/.local/lib/python2.7/site-packages/ansibleinventorygrapher/inventory.py", line 158, in __init__
    self.inventory = INVENTORY(inventory, ask_vault_pass, vault_password_file)
  File "/home/user/.local/lib/python2.7/site-packages/ansibleinventorygrapher/inventory.py", line 104, in __init__
    super(Inventory24, self).__init__(inventory, ask_vault_pass, vault_password_file)
  File "/home/user/.local/lib/python2.7/site-packages/ansibleinventorygrapher/inventory.py", line 24, in __init__
    self.vault_pass = self.ask_vault_password()
  File "/home/user/.local/lib/python2.7/site-packages/ansibleinventorygrapher/inventory.py", line 98, in ask_vault_password
    from ansible.parsing.vault import PromptVaultPass
ImportError: cannot import name PromptVaultPass

 $ pip list | grep ansible
ansible (2.4.1.0)
ansible-inventory-grapher (2.4.2)

**Comment by willthames** [comment]
I've just pushed an attempt at the fix to the vault_fix branch

You can test from source with

```
cd ansible-inventory-grapher
git checkout vault_fix
PYTHONPATH=lib:$PYTHONPATH bin/ansible-inventory-grapher ...
```

**Comment by mf69** [comment]
For me...
It works with --ask-vault-pass

It doesn't work with --vault-password-file
...
TypeError: unbound method _read_file() must be called with FileVaultSecret instance as first argument (got str instance instead)

**Comment by willthames** [comment]
Fixes pushed to v2.4.3
